src/routes/Wishlist.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import WishlistProductCard from "../components/wishlist-page/WishlistProductCard";
+
+const product = {
+  newArrival: true,
+  image: "https://picsum.photos/500",
+  title: "Some Title",
+  description:
+    "lorem ipsum dolor sit lorem ipsum dolor sitlorem ipsum dolor sitlorem ipsum dolor sitlorem ipsum dolor sit",
+  actualPrice: 1000,
+  discountedPrice: 900,
+  qty: 1,
+};
+
+function Wishlist() {
+  return (
```

Comment:
Suggestion : implementation of APIs needed for fetching data from backend

Reply:
yes ill havent made api calls yet